Docs: replacing "setting" to "option" in world docs (#2622)
ArchipelagoMW/Archipelago: worlds/witness/docs/setup_en.md

3. Click on the "AP" symbol at the top.
4. Enter the AP address, slot name and password. 

> The rest should take care of itself! Items and checks will be marked automatically, and it even knows your options - It will hide checks & adjust logic accordingly.

Reviewer: Ugh. Yeah, another instance where I'm really frustrated by the ambiguity of the word "option". But this one is more acceptable. I would personally probably also reword this to say something like "the options you picked", but it's not as necessary imo

Author: ```suggestion
The rest should take care of itself! Items and checks will be marked automatically, and it even knows the options you chose - It will hide checks & adjust logic accordingly.
```

Reviewer: Yeah I like that :3

> The rest should take care of itself! Items and checks will be marked automatically, and it even knows the options you chose - It will hide checks & adjust logic accordingly.

Author: Might revert?

> The rest should take care of itself! Items and checks will be marked automatically, and it even knows the options you chose - It will hide checks & adjust logic accordingly.

Author: ```suggestion
The rest should take care of itself! Items and checks will be marked automatically, and it even knows your settings - It will hide checks & adjust logic accordingly.
```
